docs/31-long-running-job.md

```diff
@@ -0,0 +1,83 @@
+# Long Running Job Management
+
+The doAzureParallel package allows you to manage long running jobs easily. There are 2 ways to run a job:
+- Synchronous
+- Asynchronous
+
+Long running job should run in asynchronous mode.
+
+## How to configure a job to run asynchronously
+You can configure a job to run asynchronously by specifying wait = FALSE in job options:
+
+```R
+  options <- list(wait = FALSE)
+  jobId <- foreach(i = 1:number_of_iterations, .options.azure = options) %dopar% { ... }
+```
+The returned value is the job ID associated with the foreach loop. Use this returned value you can get job status and job result.
+
+## Get job status
+
+getJob returns job metadata, such as chunk size, whether cloud combine is enabled, and packages specified for the job, it also returns task acount in different state
+
+```R
+  getJob(jobId)
+  getJob(jobId, verbose = TRUE)
```

Comment:
Does the verbose print these values out?

What's the output for getJob(jobId)

Reply:
verbose is TRUE by default, so the output for both lines are the same, if this is not desired, we can set verbose to FALSE by default